src/events/crawler/lib/stringify.js

```diff
@@ -29,8 +29,9 @@ export const csvForDay = function(data) {
     }
   }
 
-  // Drop coordinates
-  columns = columns.filter(column => column !== 'coordinates');
+  // Drop unwanted columns.
+  const unwanted = ['coordinates', 'maintainers'];
+  columns = columns.filter(column => !unwanted.includes(column));
```

Comment:
What about curators/sources? They're not coming through?

Reply:
According to crawler/lib/stringify.js, no.  The first columns (up to `url`) are included by default, and then the code adds extra columns according to what's in the data being written.

Reply:
Note: the headings shown could just be due to the way that I ran my manual test, using only one county.  I didn't generate it using all locations.